src/components/SwitzerlandEstimatedCasesDivisionModal.tsx

```diff
@@ -87,26 +88,23 @@ export const SwitzerlandEstimatedCasesDivisionModal = ({
   show,
   handleClose,
 }: Props) => {
-  const [caseCounts, setCaseCounts] = useState<CaseCountEntry[] | undefined>();
+  //const [caseCounts, setCaseCounts] = useState<CaseCountEntry[] | undefined>();
```

Comment:
We can avoid saving the response data in a local state (check [blog post](https://blog.bitsrc.io/how-to-start-using-react-query-4869e3d5680d) )